### PR TITLE
change title in base.html and message file

### PR DESCRIPTION
**Pull request by harikrishnagit**
[Link to ZenHub issue.](link-goes-here)

## What does this change?
Change base page title to include "contact the " 

### Author

+ [x
Screen Shot 2020-07-01 at 5 40 44 PM
 ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1183,8 +1183,8 @@ msgid "Any supporting materials (please list and describe them)."
 msgstr "Cualquier material de apoyo (haga una lista y describa cada artículo)."
 
 #: templates/base.html:22 templates/base.html:40
-msgid "Civil Rights Division - Department of Justice"
-msgstr "La División de Derechos Civiles - Departamento de Justicia"
+msgid "Contact the Civil Rights Division | Department of Justice"
+msgstr "Comuníquese con la División de Derechos Civiles - Departamento de Justicia"
```
Comment (Jkrzy) [inline crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:1187]: We should use `|` consistently 
```suggestion
msgstr "Comuníquese con la División de Derechos Civiles | Departamento de Justicia"
```

Comment (harikrishnagit) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:1187]: Ha. yes, i just copied from the translation.

Comment (harikrishnagit) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:1187]: Done, also at report_base.html

**Comment by cantsin** [comment]
@harikrishnagit is this ready for review?

**Comment by harikrishnagit** [comment]
> @harikrishnagit is this ready for review again?

Yes, it is after handling dups

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1182,9 +1182,11 @@ msgstr ""
...
 msgstr "Cualquier material de apoyo (haga una lista y describa cada artículo)."
 
-#: templates/base.html:22 templates/base.html:40
-msgid "Civil Rights Division - Department of Justice"
-msgstr "La División de Derechos Civiles - Departamento de Justicia"
+#: templates/base.html:22 templates/base.html:40 templates/forms/base.html:19
+#: templates/forms/report_base.html:13
+msgid "Contact the Civil Rights Division | Department of Justice"
+msgstr ""
+"Comuníquese con la División de Derechos Civiles - Departamento de Justicia"
```
Comment (cantsin) [inline crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:1189]: We want to change the `-` here to `|` too (spanish version)

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1509,11 +1511,16 @@ msgid "completed"
...
+#| "Civil Rights Division - Department of Justice\n"
+#| "        "
 msgid ""
 "\n"
 "          Step %(stage_number)s: %(current_step_name)s - Contact the Civil "
-"Rights Division - Department of Justice\n"
+"Rights Division | Department of Justice\n"
 "        "
 msgstr ""
 "\n"
```
Comment (cantsin) [inline crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:1526]: Same below, we want to change `-` to `|` for spanish version

**Comment by cantsin** [comment]
@harikrishnagit checking in -- anything I can do to help here? It'd be nice to get this PR/issue off the board.

**Comment by harikrishnagit** [comment]
I think this is ready to be merged and tested. Lmk if you are merging the pull req.

**Comment by cantsin** [comment]
@harikrishnagit the changes I requested have not yet gotten in?

**Comment by harikrishnagit** [comment]
@cantsin fixed one missing |, it is ready for your review.

File: crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po (hunk 2 above)
Comment (cantsin) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po:1189]: @harikrishnagit ^

Comment (harikrishnagit) [reply crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po]: Sorry.. I keep missing one everytime. I think this is complete now. please review

**Comment by harikrishnagit** [comment]
@cantsin please review, and lmk if I can merge. @hbillings, can you review, looks like James is busy.

**Comment by cantsin** [comment]
Thank you! LGTM

**Pull request merged**